DXMainClient/DXGUI/Multiplayer/ChatListBox.cs

```diff
@@ -14,11 +16,13 @@ namespace DTAClient.DXGUI.Multiplayer
     /// </summary>
     public class ChatListBox : XNAListBox, IMessageView
     {
+        private Regex _regexp = new Regex(ClientConfiguration.Instance.AlwaysTrustedLinksRegExp);
```

Comment:
use a meaningful name (e.g., `alwaysTrustedLinksRegExp`) instead of `_regexp`.

Can you describe the purpose to separate these two trusted lists?

Also, maybe we can make both trusted sources as regular comma-separated lists?